packages/odata-service-writer/src/index.ts

```diff
@@ -89,6 +89,9 @@ async function generate(basePath: string, data: OdataService, fs?: Editor): Prom
         proxyLocalMiddleware.config,
         proxyLocalMiddleware.comments
     );
+
+    const mwMock = getMockServerMiddlewareConfig(data);
+    await addMiddlewareConfig(fs, basePath, 'ui5-local.yaml', mwMock);
```

Comment:
```suggestion
    const mockserverMiddleware = getMockServerMiddlewareConfig(data);
    await addMiddlewareConfig(fs, basePath, 'ui5-local.yaml', mockserverMiddleware);
```
Use a more descriptive name? I can't use the suggestion to capture all the usages of the variable, but you get the picture.

Reply:
Also, don't we do this only for v4 services?

Reply:
One more clarification: don't we do this conditionally in tools-suite? I see a check for a v4 service there.

Reply:
Thanks @IainSAP